frontend/src/components/annotation/annotation_core.vue

```diff
@@ -7762,6 +7763,7 @@ export default Vue.extend({
           const new_image = await this.addImageProcess(file.image.url_signed);
           this.html_image = new_image;
           this.refresh = Date.now();
+          await this.get_parent_instance_list_for_video();
```

Comment:
This was causing latency in regular use
Right now global is global to video not frame so here is ok for now
if we in future support per frame globals then would revisit
(from discussion with Pablo)